src/lib/chips/chip-input.ts

```diff
@@ -124,8 +148,18 @@ export class MatChipInput {
   /** Checks to see if the (chipEnd) event needs to be emitted. */
   _emitChipEnd(event?: KeyboardEvent) {
     if (!this._inputElement.value && !!event) {
-      this._chipList._keydown(event);
+      const isRtl = this._dir && this._dir.value === 'rtl';
+      const isPreviousKey = event &&
```

Comment:
I think the parens are wrong on this expression. I would change to:

`event && event.keyCode === (isRtl ? RIGHT_ARROW : LEFT_ARROW);`